pkg/api/types.go

```diff
@@ -1801,6 +1801,9 @@ type PodSpec struct {
 	// ServiceAccountName is the name of the ServiceAccount to use to run this pod
 	// The pod will be allowed to use secrets referenced by the ServiceAccount
 	ServiceAccountName string
+	// AutomountServiceAccountToken indicates whether a service account token should be automatically mounted.
+	// +optional
+	AutomountServiceAccountToken *bool
```

Comment:
I had hoped to harmonize this with a more general-purpose mechanism for expressing explicit service dependencies and/or explicit pod-injection, but I don't see a clear way to do that